src/Input/Textarea/Textarea.tsx

```diff
@@ -9,133 +9,116 @@ import {
   TextareaLabel,
 } from '../../Style/Input/TextareaStyle';
 
-class Textarea extends React.PureComponent<Props, State> {
-  textareaInputRef: React.RefObject<HTMLTextAreaElement>;
-
-  state = {
-    floating: false,
-    rows: 4,
-    minRows: 4,
-    maxRows: 12,
-    textareaMaxHeight: 0,
-  };
-
-  constructor(props: Props) {
-    super(props);
-    this.textareaInputRef = props.forwardedRef || React.createRef();
-  }
-
-  handleFocusChange = (
-    onBlur: (e: React.FocusEvent<HTMLTextAreaElement>) => void
-  ) => {
-    const listener = (e: React.FocusEvent<HTMLTextAreaElement>) => {
-      this.setState({
-        floating: e.target.value.length > 0,
-      });
-
-      if (onBlur !== undefined) {
-        return onBlur(e);
-      }
-    };
+const Textarea: React.FunctionComponent<Props> = props => {
+  const {
+    label,
+    value,
+    status,
+    disabled,
+    className,
+    onBlur,
+    onChange,
+    removeFloatingLabel,
+    ...restProps
+  } = props;
 
-    return listener;
-  };
+  const [floating, setFloating] = React.useState(false);
```

Comment:
Should add type check for `useState`, e.g. `React.useState<boolean>`

Reply:
Oh cool I didn't know about this. Thanks for letting me know. Added this.